generation/diff_files.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#run this script from google-cloud-java directory
+# google-cloud-java$ ./generation/diff_files.sh
+# not using set -e because failing diff command is ok
+# latest repositories Id -> comgooglecloud-5570, comgoogleapi-5038, comgoogleanalytics-1052
+# this script must be run on a branch which does not have any snapshot versions in it
+# Run the stage job for google-cloud-java, on a branch which does not have any snapshot versions in it.
+# search the stage job logs for the latest-repo-ids and edit them below under `cloudRepoId`, `apiRepoId`, `analyticsRepoId`
```

Comment:
Can you describe these as steps?

Step 1 . Run ...
Step 2. Check log for ...
Step 3. Update xxx
Step 4. Run diff_files.sh